core/src/main/java/com/linecorp/armeria/server/docs/JsonSchemaGenerator.java

```diff
@@ -199,7 +199,8 @@ private void generateMapFields(ObjectNode fieldNode, FieldInfo field, Map<String
         final ObjectNode additionalProperties = mapper.createObjectNode();
         final String nextPath = path + '/' + field.name();
 
-        // TODO: How does map<int, ...> serialze to json? I guess it uses string keys.
+        // TODO: How does map<int, ...> serialize to json? I guess it uses string keys.
+        //  Same question applies to other types too, how about Map<Object, ...>?
```

Comment:
I think we can just ignore the maps whose keys are not string. 😉

Reply:
From proto's website: https://developers.google.com/protocol-buffers/docs/proto3#json


> All keys are converted to strings.

I think it is fine then. Probably same behaviour applies to other languages as well? And this PR mainly focuses on gRPC so... Let me remove the TODO

Reply:
Is there any other important stuff we are lacking? It could be nice if this finally gets merged before 2023 😆